falcon/response.py

```diff
@@ -457,6 +466,9 @@ def set_cookie(
         .. _Same-Site RFC Draft:
             https://tools.ietf.org/html/draft-ietf-httpbis-rfc6265bis-03#section-4.1.2.7
 
+        .. _Partitioned RFC Draft:
+            https://datatracker.ietf.org/doc/html/draft-cutler-httpbis-partitioned-cookies#section-2.1
```

Comment:
Maybe we could instead link to the mdn docs? https://developer.mozilla.org/en-US/docs/Web/Privacy/Privacy_sandbox/Partitioned_cookies

Reply:
Yes, I agree that the mdn doc is a lot more helpful